### DISCUSSION TITLE
Molecule with Compose for Web

### DISCUSSION THREAD

**Discussion by kaivol** [cashapp/molecule#72] [Q&A] [answered]
If I understand correctly, Compose for Web depends on Kotlin Multiplatform but Molecule is incompatible with Multiplatform (#2), so I can't use Molecule together with Compose for Web.
Is this correct or are there any workarounds that I could use?

**Comment by JakeWharton** [comment, marked as answer]
You can copy the source into your codebase for now to use with Compose for Web or Compose for Desktop for now. The linked issue tracks support for non-Android platforms.
